src/isPalindrome.js

```diff
@@ -0,0 +1,9 @@
+export default function isPalindrome(input){
+	if (typeof input ==='string'){
+		const alphaNumeric = input.toLowerCase().replace(/[^\w]|_/g, '')
+		const reverse = alphaNumeric.split('').reverse().join('')
+		return reverse === alphaNumeric
+	}else{
+		return 'Invalid input'
```

Comment:
this check does not seem to work. I inputted numbers and the condition did not get returned